content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1006,6 +1006,50 @@ status:
   terminating: 3 # three Pods are terminating and have not yet reached the Failed phase
 ```
 
+### Delegation of managing a Job object to external controller
+
+{{< feature-state for_k8s_version="v1.30" state="alpha" >}}
+
+{{< note >}}
+You can only set `ManagedBy` field on Jobs if you enable the `JobManagedBy`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+(disabled by default).
+{{< /note >}}
+
+This feature allows you to disable the built-in Job controller, for a specific
+Job, and delegate reconciliation of the Job to an external controller.
+
+By setting a custom value of the `spec.managedBy` field - any value
```

Comment:
You indicate the controller that reconciles the Job by setting a custom value for....

Reply:
Done